tiled/server/router.py

```diff
@@ -604,14 +604,65 @@ async def table_full(
     response_model=schemas.Response,
     name="full 'container' metadata and data",
 )
-async def container_full(
+async def get_container_full(
     request: Request,
     entry=SecureEntry(scopes=["read:data"]),
     principal: str = Depends(get_current_principal),
     field: Optional[List[str]] = Query(None, min_length=1),
     format: Optional[str] = None,
     filename: Optional[str] = None,
     serialization_registry=Depends(get_serialization_registry),
+):
+    """
+    Fetch the data for the given container via a GET request.
+    """
+    return await container_full(
+        request=request,
+        entry=entry,
+        principal=principal,
+        field=field,
+        format=format,
+        filename=filename,
+        serialization_registry=serialization_registry,
+    )
+
+
+@router.post(
+    "/container/full/{path:path}",
+    response_model=schemas.Response,
+    name="full 'container' metadata and data",
+)
+async def post_container_full(
+    request: Request,
+    entry=SecureEntry(scopes=["read:data"]),
+    principal: str = Depends(get_current_principal),
+    field: Optional[List[str]] = Body(None, min_length=1),
```

Comment:
Presumably `field` will be removed from the `POST` variant.

Reply:
In the POST route, `field` is now bound to the request Body rather than a Query parameter.  I could rename it to `body` if you think that would add clarity.

Let me know if you meant something else.